cluster/saltbase/salt/kubelet/default

```diff
@@ -15,6 +15,15 @@
   {% set api_servers = "--api_servers=https://" + ips[0][0] + ":6443" -%}
 {% endif -%}
 
+# Disable registration for the kubelet running on the master on GCE.
```

Comment:
Why disable registration of the master kubelet? The API server reports the master pods but the usual APIs used to access the pods, like '/containerLogs', '/exec', etc, are not working as of now.